### DISCUSSION TITLE
Geocoding get the zoom

### DISCUSSION THREAD

**Discussion by ohadcohen1111** [osm-search/Nominatim#2852] [Q&A]
I saw that given a search in OSM I get the zoom according to the place I was searched.
Is it possible to get the zoom in response or get it in another way?

For example, my query is:
https://nominatim.openstreetmap.org/search?q=London&format=json&limit=1

**Comment by mtmail** [comment]
Nominatim doesn't return zoom levels.

In the JSON you'll find a `boundingbox`. With a leafletjs powered map, like www.openstreetmap.org uses it, you can use `map.fitBounds` to make sure London is centered and shows it's extend. https://leafletjs.com/reference.html#map-fitbounds

Alternatively you'd need to do some math to calculate a good zoom level based on the bounding box coordinates. But that would a more generic (non-Nominatim) question for https://help.openstreetmap.org/ or https://gis.stackexchange.com/
